.travis.yml

```diff
@@ -2,14 +2,16 @@ language: php
 
 # Workaround for fixing that Selenium server is not running and therefore Behat tests not working
 sudo: required
+# ORIGINAL:
+# sudo: false
 
 addons:
   firefox: "47.0.1"
   postgresql: "9.3"
   apt:
     packages:
-      - oracle-java9-installer
-      - oracle-java9-set-default
+      - oracle-java8-installer
```

Comment:
You downgraded java

Reply:
It's a mistake sorry^^.. I had a conflict and when resolving it I deleted java9 lines instead of java8.
I'm gonna fix it and push it again.